modules/tests/test/testconfigs/copy-template-config.go

```diff
@@ -32,28 +27,15 @@ type CopyTemplateTestConfig struct {
 }
 
 func (c *CopyTemplateTestConfig) Init(options *testoptions.TestOptions) {
-	c.TaskData.ResultTemplateName = NewTemplateName
-
-	if c.TaskData.SourceTemplateName == "" && !c.TaskData.DoNotSetSourceName {
-		c.TaskData.SourceTemplateName = testtemplate.CirrosTemplateName
-	}
-
-	if c.TaskData.TargetTemplateName == "" && !c.TaskData.DoNotSetTargetName {
-		c.TaskData.TargetTemplateName = NewTemplateName
-	}
-
 	c.deploymentNamespace = options.DeployNamespace
 
-	c.TaskData.SourceNamespace = options.ResolveNamespace(c.TaskData.SourceTemplateNamespace, options.TestNamespace)
+	c.TaskData.SourceNamespace = options.ResolveNamespace(c.TaskData.SourceTemplateNamespace, options.DeployNamespace)
```

Comment:
originally having deploy/test namespace was done to test RBAC permissions. That is why we have tests: cluster-scope and namespace scope (where deploy == test)

so for the cluster scope we want to create taskruns in deploy ns and the entities they create in test ns. 
Because of that we should default to TestNamespace in our entities to test accross namespaces by default. 

Then we can have one or few tests that test the edge cases. Like creating entities in deploy namespace as mentioned here: https://github.com/kubevirt/kubevirt-tekton-tasks/pull/93#discussion_r708398066

Reply:
Although I am not sure if it still make sense to split it like that and have separate jobs for 2 sets of RBACs since it doesn't test the code but the deployment. Originally it wasn't clear if we would distribute two yamls (one for namespace scope and one for cluster scope) or just one, so these jobs were testing that. But since then we switched to distributing only the namespace scope and leaving the rest to the user: https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/docs/tasks-rbac-permissions.md